CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [unreleased]
+
+### Added
+- `UIConfig.enterFullscreenOnInitialPlayback` to enter fullscreen when clicking `PlaybackToggleButton` or `HugePlaybackToggleButton` to initiate the initial playback
```

Comment:
what happens if the user has autoplay enabled, the flag `enterFullscreenOninitalPlayback` set to true, then pauses and plays again. Will it enter fullscreen?

Reply:
No, because the `firstPlay` flag is changed in the `Play` event listener: https://github.com/bitmovin/bitmovin-player-ui/blob/d3f7714a4224cdd05955d2277f9cbb76ff7cc844/src/ts/components/playbacktogglebutton.ts#L64-L66

Reply:
👍